Userland/Applications/Piano/ProcessorParameterWidget/Slider.h

```diff
@@ -6,19 +6,19 @@
 
 #pragma once
 
+#include "WidgetWithLabel.h"
 #include <LibDSP/ProcessorParameter.h>
 #include <LibGUI/Label.h>
 #include <LibGUI/Slider.h>
 #include <LibGfx/Orientation.h>
 
-class ProcessorParameterSlider : public GUI::Slider {
+class ProcessorParameterSlider : public GUI::Slider
+    , public WidgetWithLabel {
```

Comment:
Please format this as

```suggestion
class ProcessorParameterSlider
    : public GUI::Slider
    , public WidgetWithLabel {
```